lib/mavenlink/client.rb

```diff
@@ -8,6 +8,9 @@ def initialize(settings = Mavenlink.default_settings)
       @oauth_token = settings[:oauth_token] or raise ArgumentError, 'OAuth token is not set'
       @endpoint = settings[:endpoint] || ENDPOINT
       @use_json = settings[:use_json]
+      if settings.key?(:custom_client_name)
+        @custom_client_name = settings[:custom_client_name]
```

Comment:
the setting should be named `user_agent_override` ... or something with the term `user_agent` (not "client name") because "client name" is too vague (is it a customer client name, or what?) "User agent" is more specific